tests/examples/opinions/united_states/bia_example.compare.json

```diff
@@ -0,0 +1,578 @@
+[
+  {
+    "case_dates": "2021-07-01",
+    "case_names": "VUCETIC",
+    "download_urls": "/eoir/file/1381766/download",
+    "precedential_statuses": "Unpublished",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": true,
```

Comment:
Why? Maybe add a comment above about this?

Reply:
I assume you mean a comment when we save it - not in the compare json file.

Reply:
Yes.

Reply:
Just to make things weird.

I tested all the compare json files.

320 Total in this branch 29 are empty;  122 have download urls with http/https:// and 169 do not have the full url.

Reply:
@arderyp Any idea why some generate full urls and some generate just the paths?

Reply:
I think this is resolved now.